assets/tasks/common.json

```diff
@@ -672,13 +731,157 @@
             ],
             "problemMatcher": "$msCompile",
             "icon": {
-                "id": "flame",
+                "id": "gear",
                 "color": "terminal.ansiYellow"
+            },
+            "presentation": {
+                "echo": true,
+                "reveal": "always",
+                "focus": true,
+                "panel": "shared",
+                "group": "build-execution"
             }
         },
         {
-            "label": "remove-dangling-images",
-            "detail": "",
+            "label": "pre-cleanup",
+            "detail": "Clean the service of the application on the target (default device)(does not down the dependencies)",
+            "hide": true,
+            "command": "docker",
+            "type": "shell",
+            "args": [
+                "compose",
+                "-p",
+                "torizon",
+                "stop",
+                "__container__",
+                "&&",
+                "docker",
+                "compose",
+                "-p",
+                "torizon",
+                "rm",
+                "-f",
+                "__container__"
+            ],
+            "dependsOrder": "sequence",
+            "dependsOn": [
+                "remove-dangling-images",
+                "wait-a-bit"
+            ],
+            "problemMatcher": "$msCompile",
+            "icon": {
+                "id": "flame",
+                "color": "terminal.ansiYellow"
+            }
+        },
+        {
+            "label": "pre-cleanup-debug",
```

Comment:
Tested here on imx6ull and it seems that it is not bringing down the other containers, which is leading to problems
![image](https://github.com/user-attachments/assets/4fad7b4e-7006-41b3-828b-873578380107)

Reply:
This is by design, now it only down the service of the workspace application. This is yet another change to make the development flow faster between one deployment and another. What types of problems are you facing?

Reply:
It is not bringing down the other containers before running my container, slowing down the execution of the container

Reply:
It would be nice if we could figure out a way to run the previous command (which is called now `containers-cleanup` I think) at least the first time the person starts a debug session, so it starts with all other containers down and also removes the orphan containers

Reply:
Please keep in mind that you are testing different stacks, this is not the field scenario where the user sticks in a workspace with a specific stack. 

Your user case is from a template developer/tester, so if run this makes sense so your scenario add it to the test pipeline.